ethereum/retesteth/src/main/java/org/hyperledger/besu/ethereum/retesteth/RetestethContext.java

```diff
@@ -206,7 +206,7 @@ private boolean buildContext(
             retestethClock,
             metricsSystem,
             syncState,
-            Wei.ZERO,
+            new MiningParameters.Builder().build(),
```

Comment:
everywhere else you've replaced this with
            new MiningParameters.Builder().minTransactionGasPrice(Wei.ZERO).build(),

Reply:
Ahh yes, was attempting to be explicit (the default is Wei.ZERO)